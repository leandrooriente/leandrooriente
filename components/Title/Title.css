.title {
  font-family: var(--headline-font-family);
  font-size: var(--headline-font-size);
  line-height: 1.6;
}

@media screen and (max-width: 468px) {
  .title {
    font-size: var(--headline-font-size-mobile);
  }
}