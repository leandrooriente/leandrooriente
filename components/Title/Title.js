import React from 'react';
import PropTypes from 'prop-types';
import css from './Title.css';

const Title = ({ children, className }) => {
  const classes = `${css.title}${className ? ` ${className}` : ''}`;
  return (
    <h1 className={classes}>
      { children }
    </h1>
  );
};

Title.defaultProps = {
  className: null,
};

Title.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node.isRequired,
};

export default Title;
