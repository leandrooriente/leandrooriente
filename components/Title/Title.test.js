import React from 'react';
import { mount } from 'enzyme';
import Title from './Title';

describe('<Title />', () => {
  it('should render <Title /> component', () => {
    const wrapper = mount(<Title>Test</Title>);

    expect(wrapper).toMatchSnapshot();
  });
});
