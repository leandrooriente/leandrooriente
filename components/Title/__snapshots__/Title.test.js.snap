// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render <Title /> component 1`] = `
<Title
  className={null}
>
  <h1
    className="title"
  >
    Test
  </h1>
</Title>
`;
